### PR TITLE
Version 3.5.2

### PR DESCRIPTION
**Pull request by UselessGuru**
Changes as in 
https://github.com/MultiPoolMiner/MultiPoolMiner/releases/tag/3.5.2
https://github.com/MultiPoolMiner/MultiPoolMiner/releases/tag/3.5.1
https://github.com/MultiPoolMiner/MultiPoolMiner/releases/tag/3.5.0
https://github.com/MultiPoolMiner/MultiPoolMiner/releases/tag/3.4.8

### CODE REVIEW

File: Balances/Nicehash.ps1 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-using module ..\Include.psm1
+﻿using module ..\Include.psm1
```
Comment (aaronsace) [inline Balances/Nicehash.ps1]: Double check that the encoding is correct.

File: Balances/NicehashExternal.ps1 [hunk 2]
```diff
@@ -0,0 +1,77 @@
+﻿using module ..\Include.psm1
```
Comment (aaronsace) [inline Balances/NicehashExternal.ps1]: Double check that the encoding is correct.

File: Devices.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{
+﻿{
```
Comment (aaronsace) [inline Devices.txt]: Double check that the encoding is correct.

File: Algorithms.txt [hunk 4]
```diff
@@ -1,4 +1,4 @@
-{
+﻿{
```
Comment (aaronsace) [inline Algorithms.txt]: Double check that the encoding is correct.

File: Get-Binaries.ps1 [hunk 5]
```diff
@@ -1,4 +1,4 @@
-using module .\Include.psm1
+﻿using module .\Include.psm1
```
Comment (aaronsace) [inline Get-Binaries.ps1]: Double check that the encoding is correct.

File: CoinNames.txt [hunk 6]
```diff
@@ -1,4 +1,4 @@
-{
+﻿{
```
Comment (aaronsace) [inline CoinNames.txt]: Double check that the encoding is correct.

File: Balances/Nicehash.ps1 (hunk 1 above)
Comment (UselessGuru) [reply Balances/Nicehash.ps1]: ï»¿Set-Location 

Encoding **IS** UTF-8 with BOM

Comment (UselessGuru) [reply Balances/Nicehash.ps1]: Encoding IS UTF-8 with BOM

File: Balances/NicehashExternal.ps1 (hunk 2 above)
Comment (UselessGuru) [reply Balances/NicehashExternal.ps1]: ï»¿using module 

Encoding IS UTF-8 with BOM

File: Devices.txt (hunk 3 above)
Comment (UselessGuru) [reply Devices.txt]: ï»¿{
    "amd":

Encoding IS UTF-8 with BOM

File: Algorithms.txt (hunk 4 above)
Comment (UselessGuru) [reply Algorithms.txt]: ï»¿{
    "aeon"

Encoding IS UTF-8 with BOM

File: Get-Binaries.ps1 (hunk 5 above)
Comment (UselessGuru) [reply Get-Binaries.ps1]: ï»¿using module 

Encoding IS UTF-8 with BOM

File: CoinNames.txt (hunk 6 above)
Comment (UselessGuru) [reply CoinNames.txt]: ï»¿{
    "01": 

Encoding IS UTF-8 with BOM

**Pull request merged**